com.unity.render-pipelines.core/ShaderLibrary/ImageBasedLighting.hlsl

```diff
@@ -588,39 +583,15 @@ real4 IntegrateLDCharlie(TEXTURECUBE_PARAM(tex, sampl),
         real D = D_Charlie(NdotH, roughness);
         real Vis = V_Charlie(NdotL, NdotV, roughness);
 
-        real mipLevel = 0;
-        if (prefilter) // Prefiltered BRDF importance sampling
-        {
-            // Use lower MIP-map levels for fetching samples with low probabilities
-            // in order to reduce the variance.
-            // Ref: http://http.developer.nvidia.com/GPUGems3/gpugems3_ch20.html
-            //
-            // - OmegaS: Solid angle associated with the sample
-            // - OmegaP: Solid angle associated with the texel of the cubemap
-
-            real omegaS;
-
-            // real PDF = 1.0f / NdotL
-            // Since (N == V), NdotH == LdotH.
-            real pdf = 1.0 /  NdotL;
-            // TODO: improve the accuracy of the sample's solid angle fit for GGX.
-            omegaS    = rcp(sampleCount) * rcp(pdf);
-
-            // 'invOmegaP' is precomputed on CPU and provided as a parameter to the function.
-            // real omegaP = FOUR_PI / (6.0 * cubemapWidth * cubemapWidth);
-            const real mipBias = roughness;
-            mipLevel = 0.5 * log2(omegaS * invOmegaP) + mipBias;
-        }
-
         // TODO: use a Gaussian-like filter to generate the MIP pyramid.
         real3 val = SAMPLE_TEXTURECUBE_LOD(tex, sampl, L, mipLevel).rgb;
 
         // Use the approximation from "Real Shading in Unreal Engine 4": Weight ~ NdotL.
-        lightInt +=  val * F * D * Vis;
+        lightInt += val * F * D * Vis;
         cbsdfInt += F * D * Vis;
     }
 
-    return real4(lightInt / cbsdfInt, 1.0);
+    return 0.5f * real4(lightInt / cbsdfInt, 1.0);
```

Comment:
can you add a comment to say that the 0.5 improve the result compare to the reference but we don't know exactly why :)

Reply:
I'm gonna try and see if I can understand why we'd need this.

Reply:
Looks like the reference model was missing pi factor, so now the error is pi/2 instead of 0.5 :) Needs to be investigated where this error is coming from